### PR TITLE
Feature/news reader

### PR DESCRIPTION
**Pull request by iankooo**

### CODE REVIEW

File: app/src/main/java/com/schipala_ianko_radoslav/newsreader/feature/newsreaderlist/MainActivity.java [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
 
 import android.os.Bundle;
 
-import com.schipala_ianko_radoslav.newsreader.ui.main.MainFragment;
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.schipala_ianko_radoslav.newsreader.R;
+import com.schipala_ianko_radoslav.newsreader.ui.main.NewsListFragment;
 
 public class MainActivity extends AppCompatActivity {
```
Comment (Misca) [inline app/src/main/java/com/schipala_ianko_radoslav/newsreader/feature/newsreaderlist/MainActivity.java]: let's rename this activity

File: app/src/main/java/com/schipala_ianko_radoslav/newsreader/feature/newsreaderlist/model/NewsReaderViewModel.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+                .map(new ArticlesToVMListMapper())
+                .observeOn(AndroidSchedulers.mainThread())
+                .subscribe(
+                        this::onNewsArticlesReceived,
+                        this::onNewsArticlesError
+                );
+    }
+
+    private void onNewsArticlesReceived(List<ArticleItemViewModel> articles) {
+        //isLoading.set(false);
```
Comment (Misca) [inline app/src/main/java/com/schipala_ianko_radoslav/newsreader/feature/newsreaderlist/model/NewsReaderViewModel.java]: let's cleanup or start using these

**Review by Misca — CHANGES_REQUESTED** [review]
good job, just a couple of changes away

File: app/src/main/java/com/schipala_ianko_radoslav/newsreader/feature/newsreaderlist/MainActivity.java (hunk 1 above)
Comment (iankooo) [reply app/src/main/java/com/schipala_ianko_radoslav/newsreader/feature/newsreaderlist/MainActivity.java]: done

File: app/src/main/java/com/schipala_ianko_radoslav/newsreader/feature/newsreaderlist/model/NewsReaderViewModel.java (hunk 2 above)
Comment (iankooo) [reply app/src/main/java/com/schipala_ianko_radoslav/newsreader/feature/newsreaderlist/model/NewsReaderViewModel.java]: done

**Review by Misca — APPROVED** [review]
GG 👍

**Pull request merged**